ddtrace/tracer/tracer.go

```diff
@@ -70,13 +70,13 @@ type tracer struct {
 
 	// These integers track metrics about spans and traces as they are started,
 	// finished, and dropped
-	spansStarted, spansFinished, tracesDropped int64
+	spansStarted, spansFinished, tracesDropped atomic.Int64
```

Comment:
These ones I'm not sure about. It looks like they're all re-set, but how often? Would a really busy app that creates tons of spans make these counters overflow? You could make them `*atomic.Int64` and initialize them with `atomic.NewInt64` in `newUnstartedTracer`.

Reply:
These are sent and reset every 10 seconds, which means in order to overflow, the tracer would need to be sending 
2<sup>31</sup> / 10 ~= 214,000,000 spans/s which is far beyond what we are capable of or what is reasonable.